git-fix-branch

```diff
@@ -7,7 +7,7 @@ source "${BASH_SOURCE%/*}/strict-mode.sh"
 
 export FILTER_BRANCH_SQUELCH_WARNING=1
 
-parent=${1-master}
+parent=${1-$(git config init.defaultBranch)}
```

Comment:
@dkubb It appears this is not set for all repositories. I've got repositories that print the empty string on that command. 

I suspect that `git-tools` should default to master after reading an empty string from that config variable.